src/background/controller/provider/controller.ts

```diff
@@ -595,75 +684,54 @@ class ProviderController extends BaseController {
             'eth_sendRawTransaction',
             [rawTx]
           );
+          try {
+            openapiService.traceTx(
+              hash!,
+              traceId || '',
+              chainItem?.serverId || ''
+            );
+          } catch (e) {
+            // DO nothing
+          }
```

Comment:
```suggestion
```
remove this, FYI: https://github.com/RabbyHub/Rabby/commit/c2ccce57d27c37fbb1f3b493442a27a7f3b6be1e#diff-8b4639172afdc1cf22ddb09ec5ef9d896214d458eacab485742333ec0dd668db